sys/fmt/fmt.c

```diff
@@ -83,6 +83,16 @@ size_t fmt_strlen(const char *str)
     return (tmp - str);
 }
 
+size_t fmt_strnlen(const char *str, size_t maxlen)
+{
+    const char *tmp = str;
+    const char *end = str + maxlen;
```

Comment:
```
size_t fmt_strnlen(const char *str, size_t maxlen)
{
    const char *tmp = str;
    while(*tmp && maxlen--) {
        tmp++;
    }
    return (tmp - str);
}
```

... would more fit the style of the rest of the file.

Reply:
Agreed, fixed